jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ToolkitAddConnectionDialog.kt

```diff
@@ -44,19 +44,20 @@ data class ConnectionDialogCustomizer(
     val title: String? = null,
     val header: String? = null,
     val helpId: HelpIds? = null,
-    val replaceIamComment: String? = null,
+    val replaceIamComment: String? = null
 )
 
 open class ToolkitAddConnectionDialog(
     private val project: Project,
     connection: ToolkitConnection? = null,
-    private val customizer: ConnectionDialogCustomizer? = null
+    private val customizer: ConnectionDialogCustomizer? = null,
+    private val addConnectionThroughCodeWhispererDialog: Boolean = false
```

Comment:
i don't think this will actually be required after the other comment, but use the customizer data class